flipperdevices/flipperzero-firmware
# Loader: good looking error messages

# What's new

- Loader now has good looking error messages in GUI
- Favourite App option now supports external menu apps

# Verification 

- Load FAP with incorrect API version
- Load an app from the main set of apps (main menu) that is not on the SD card. You should see "Update needed" error message
- Check Favourite App setting

# Checklist (For Reviewer)
